### ISSUE TITLE
Post not showing on site

### ISSUE THREAD

**Issue by ishandandekar** [cotes2020/jekyll-theme-chirpy#1133]
## Description

I followed the tutorial stated, and added a markdown file in the `_posts` directory. The site is not the post. I tried multiple methods but can't seem to find a solution.

## Operations you have already tried

I copied and added other people's markdown files to check if maybe I had missed something, but that went in vain. I have checked the issues but the solutions didn't work for me

## Logs/Screenshots
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/69683819/7c674085-fd70-4271-b1e8-e41171818480)

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/69683819/f0c8e2f6-77e4-4cd1-9a55-cdac3a90f787)

**Comment by Mohamed3nan** [comment]
I think this format is wrong
https://github.com/ishandandekar/ishandandekar.github.io/blob/c87af80a44ea735ea67083f96bd636713f3e8741/_posts/2023-07-10-First.md?plain=1#L3C1-L3C34

**Comment by ishandandekar** [comment]
Still isn't working! Did I do something wrong? Could the check the repo again, please?

**Comment by azhu021** [comment]
I rolled your repo and deployed directed on my GitHub account. It works will.
![Screenshot_20230710_192049_com microsoft emmx](https://github.com/cotes2020/jekyll-theme-chirpy/assets/115162462/7e5f378b-6c39-4108-a1a6-e9ef900ecff7)
 In your condition.I think your GitHub action may incorrect.

**Comment by ishandandekar** [comment]
I copied the `jekyll.yml` from your fork and added that in my workflows. It is working now. Hopefully this problem won't persist. But I was wondering what the problem was. If you know the answer please do let me know!

**Comment by azhu021** [comment]
Generally, Jekyll will render your content (themes files, posts, pages, etc.) to static html files. The outputs are in the _site directory. Then, your http server will regard this directory as the root directory, so visitors can see your blog via your domain. 
When I visited your URL, I saw a raw index.html file with it's source code. Obviously, your jekyll site isn't rendering correctly. So I guess you didn't set your GitHub workflow correctly.
